src/app.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+
+var todoItems = [];
```

Comment:
Global variables like this can cause problems... can you think of a way to get rid of this?